bin/specextract

```diff
@@ -3060,27 +3060,28 @@ def specextract(args):
                     
             if instrument == "ACIS":
                 try:
-                    if True in [weight == "yes", srcbkg == "bkg" and dobkgresp]: # force background to always be weighted
+                    if True in [srcbkg == "src" and weight == "yes", srcbkg == "bkg" and dobkgresp]: # force background to always be weighted
```

Comment:
It's better written

    if any([...]):

as `any` will return True if any of the elements in the input sequence are True. There's also an `all` function to require all elements of the sequence to be True, but that's not useful here.

Reply:
@DougBurke these have been revised